backend/settings.py

```diff
@@ -1,10 +1,10 @@
 import os
-from typing import Optional
+from typing import Optional,ClassVar
 
 import orjson
-from pydantic import BaseSettings
 
 from backend.types import EmbeddingCacheConfig, MetadataStoreConfig, VectorDBConfig
+from pydantic_settings import BaseSettings
 
 
 class Settings(BaseSettings):
```

Comment:
Maybe not in this PR but I think we should be able to get rid of `os.getenv`
after all pydantic settings is supposed to read from env on its own